katz/src/main/kotlin/katz/data/Cokleisli.kt

```diff
@@ -7,6 +7,8 @@ typealias CokleisiFun<F, A, B> = (HK<F, A>) -> B
 
 typealias CoreaderT<F, A, B> = Cokleisli<F, A, B>
 
+fun <F, A, B> CokleisiTKind<F, A, B>.ev(): Cokleisli<F, A, B> = this as Cokleisli<F, A, B>
```

Comment:
For consistency. At the moment the ev() is available for all HK types. If it's at the bottom you have to make sure to add methods right before it, which it's a practice that may not be followed.

With this change, just from opening the file you know your alias and casts without having to scroll.